tests/ashley/machina_extensions/forum/test_forum_view.py

```diff
@@ -364,3 +369,177 @@ def test_testing_query_param(self):
             '<a href="?o=3&whatever=2" class="toggle descending" title="Toggle sorting"></a>',
             html=True,
         )
+
+    def test_testing_topic_is_sticky_stay_sticky_default_order(self):
+        """Request page with a sticky topic and check that the sticky topic is always the first
+        result on default order"""
+        forum, url_list_topic = self._get_url_list_topic_with_three_topics()
+        # Creates a post for sticky topic
+        PostFactory(
+            topic=TopicFactory(forum=forum, type=Topic.TOPIC_STICKY),
+            subject="TOPIC STICKY ONE",
+        )
+
+        # Calls page with default order
+        url = f"{url_list_topic}"
+        response = self.client.get(url)
+
+        topicC = forum.topics.get(subject__startswith="TOPIC C the newest")
+        topicB = forum.topics.get(subject__startswith="TOPIC B the eldest")
+        topicA = forum.topics.get(subject__startswith="TOPIC A created second")
+        topicSticky = forum.topics.get(subject__startswith="TOPIC STICKY ONE")
```

Comment:
this calls for a traditional use of factories in your method because you could have caught the instances from their creation?

I would do all the models creation in the test function and use randomness on everything else including the title to make sure it's sorting on the date indeed. To illustrate my concern, your topics could be sorted on the title and it would work the same because your titles are C > B > A as well...

Reply:
Sam this has been done for the previous pr not to repeat everything every time and as a compromise.
The line below explain that it can be trusted
`self.assertTrue(topicC.last_post_on > topicA.last_post_on > topicB.last_post_on)`